.github/workflows/recipe_integration_test.yaml

```diff
@@ -65,6 +65,7 @@ jobs:
         run: |
           python -m pip install -r requirements.txt
           python -m pip install -r dev-requirements.txt
+          python -m pip install lm-eval==0.4.*
```

Comment:
How did we decide on this particular version?

Reply:
It is used by lit-gpt and gpt-fast and is the most up-to-date release, plus there would be significant "hacks" to be put in place between 0.3 and 0.4 to make it work e.g. using BaseLM instead of HFLM.